### DISCUSSION TITLE
I want to use it commercially, do I need to obtain a commercial license?

### DISCUSSION THREAD

**Discussion by Zephery** [teslamate-org/teslamate#3362] [General]
I want to integrate teslamate into my own mini program and then sell it to external parties. Do I need to obtain authorization? and How to contact?

**Comment by cwanja** [comment]
@Zephery it is entirely licensed under MIT https://github.com/adriankumpf/teslamate/blob/master/LICENSE which makes it freeware. It comes with no support other than through these channels. 

My only recommendation would be - make a small percentage of your sales and donate it back to @adriankumpf.
